src/apps/basket/forms.py

```diff
@@ -1,8 +1,6 @@
 from django import forms
 
-PRODUCT_QUANTITY_CHOICES = [
-    (i, str(i)) for i in range(1, 21)
-]  # todo add quantity validation before add
+PRODUCT_QUANTITY_CHOICES = [(i, str(i)) for i in range(1, 21)]
```

Comment:
Could you please explain the reason of range(1, 21)?

Reply:
I don`t remember already the reason as we don`t have any business requirement . It was added at the very beginning. I think the quantity depends on ourselves. So we can discuss it

Reply:
so, do you understand why it was added in such way? 
maybe we need to refactor it in case it has no logic

Reply:
yes, we have our "business limitation". As far as i understand it depends wether it wholesale or retail. Please, share if you have another idea

Reply:
currently we can't add at once more than 20 products. 
moreover, user won't het any error. so, my suggestion to get rid of it (remember YAGNI? :)

you have a validation on stock, that's the main point of the task